app/api/models.py

```diff
@@ -45,3 +45,11 @@ def check_exclusive_diagnosis_or_ctrl(cls, values):
                 detail="Subjects cannot both be healthy controls and have a diagnosis.",
             )
         return values
+
+
+class AggDatasetResponse(BaseModel):
+    """Data model for query results aggregated at the dataset-level."""
+
+    dataset: str = None
+    dataset_name: str = None
+    num_matching_subjects: int
```

Comment:
Is the number of subjects the only mandatory attribute here?

Reply:
For now, yes, since we are accounting for the case where we want to return to the user that `num_matching_subjects = 0`, in which case `dataset` and `dataset_name` wouldn't have a value.